amethyst_utils/src/ortho_camera.rs

```diff
@@ -86,6 +89,35 @@ impl Component for CameraOrtho {
     type Storage = DenseVecStorage<Self>;
 }
 
+/// CameraOrtho prefab
```

Comment:
Would be nice to have a tiny bit more doc.

Reply:
Heh, I'lll try the derive. Hadn't seen that.

Reply:
Do you think it still needs docs if it's just a derive?